crates/rpc-types-engine/src/error.rs

```diff
@@ -51,7 +51,7 @@ pub enum PayloadError {
     #[display("requests present in pre-prague payload")]
     PrePragueBlockRequests,
     /// Invalid payload block hash.
-    #[display("block hash mismatch: want {consensus}, got {execution}")]
+    #[display("block hash mismatch: want {execution}, got {consensus}")]
```

Comment:
yeah this isn't super obvious which one is which like what is want

so unsure if this is change matches got vs expected

Reply:
Agree it can be confusing. I was getting this error from Reth when sending a payload from a consensus client and, from that perspective, it was clear it was the other way around.